README.rst

```diff
@@ -20,14 +20,8 @@ of a Sphinx theme based on an earlier version of the `mkdocs-material
 significantly diverged from the upstream `mkdocs-material
 <https://squidfunk.github.io/mkdocs-material/>`__ repository.
 
-See the `TensorStore documentation <https://google.github.io/tensorstore/>`__
-for a demonstration of this theme.
-
-WARNING: This theme is currently still a work in progress.  The documentation is
```

Comment:
It is still worth reserving the right to make breaking changes without notice since we're not yet in a major stable release.

Reply:
I updated the message to note that.